lnrpc/lightning.proto

```diff
@@ -2733,6 +2736,10 @@ message PendingChannelsResponse {
 
         // The transaction id of the closing transaction
         string closing_txid = 4;
+
+        // The raw hex encoded bytes of the closing transaction. Included if
+        // include_raw_tx is true.
```

Comment:
nit: "Included if include_raw_tx in the request is true", since there is no direct relationship between request/response in the API docs.

Reply:
Done